client/stylesheets/_toCat.scss

```diff
@@ -4708,7 +4708,7 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .searchResultsCardUl > li:hover {
-    border: 1px solid #f8504b !important;
+    border: 1px solid #999 !important;
```

Comment:
Color literals like `#999` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces